### PR TITLE
Octave Demo Script

### PR DESCRIPTION
**Pull request by yuvalqm**
This is the code from the documentation page of octave.
With this script you should be able to operate a single RF output/input of the octave at a certain LO and IF.

Added the code and a configuration file to support it

### CODE REVIEW

File: Tutorials/octave/octave_demo.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.octave import *
+from qm.qua import *
+from configuration import *
+
+opx_ip = "xxx.xxx.xxx.xxx"
+opx_port = 80
+octave_ip = "xxx.xxx.xxx.xxx"
+octave_port = 80
```
Comment (yomach) [inline Tutorials/octave/octave_demo.py:11]:
```suggestion
qop_ip = "xxx.xxx.xxx.xxx"
opx_port = 80
octave_port = 80
```
We set up the devices to have the same IP but different ports. 
Of course, these changes also need to be reflected down.

File: Tutorials/octave/octave_demo.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+element = "q1"
+
+qm.octave.set_lo_frequency(element, lo_freq)
+qm.octave.set_rf_output_gain(element, -10)
+qm.octave.set_rf_output_mode(element, RFOutputMode.on)
+
+qm.octave.set_lo_source(element, OctaveLOSource.LO1)
+
+qm.octave.calibrate_element(element, [(lo_freq, if_freq)])
+qm = qmm.open_qm(config)  # Calibration closes the QM so another one should be opened again after calibration is done
```
Comment (yomach) [inline Tutorials/octave/octave_demo.py:40]: just realized that you have to open the qm twice.
very ugly :(

File: Tutorials/octave/octave_demo.py (hunk 1 above)
Comment (yuvalqm) [reply Tutorials/octave/octave_demo.py:11]: OK, I wanted it to be as general as possible in that sense

File: Tutorials/octave/octave_demo.py (hunk 2 above)
Comment (yuvalqm) [reply Tutorials/octave/octave_demo.py:40]: you need to open one for each calibration you do.. it is very ugly and sadly there was no attention to it in recent months :(

**Pull request closed without merging**